exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServerTest.java

```diff
@@ -526,4 +537,75 @@ void toBuilder() {
         .hasFieldOrPropertyWithValue("executor", executor)
         .hasFieldOrPropertyWithValue("prometheusRegistry", prometheusRegistry);
   }
+
+  /**
+   * Set the default histogram aggregation to be {@link
+   * Aggregation#base2ExponentialBucketHistogram()}. In order to validate that exponential
+   * histograms are produced, we request protobuf encoded metrics when scraping since the prometheus
+   * text format does not support native histograms. We parse the binary content protobuf payload to
+   * the protobuf java bindings, and assert against the string representation.
+   */
+  @Test
+  void histogramDefaultBase2ExponentialHistogram() throws IOException {
```

Comment:
@Abhishekkr3003 I added a test which confirms that base2 exponential histograms are in fact configured by requesting the protobuf binary format from the prometheus server, parsing it to equivalent java bindings, and asserting against its string represetnation.

Reply:
Thanks, @jack-berg , for continuing it, I have not been getting time to contribute here lately.